preflight.sh

```diff
@@ -31,4 +31,4 @@ else
   fi
 fi
 echo "::debug ::Set PREFLIGHT_SHORT_LENGTH=$PREFLIGHT_SHORT_LENGTH"
-echo "PREFLIGHT_SHORT_LENGTH$PREFLIGHT_SHORT_LENGTH" >> $GITHUB_OUTPUT
+echo "PREFLIGHT_SHORT_LENGTH=$PREFLIGHT_SHORT_LENGTH" >> $GITHUB_OUTPUT
```

Comment:
to avoid https://www.shellcheck.net/wiki/SC2086

```suggestion
echo "PREFLIGHT_SHORT_LENGTH=$PREFLIGHT_SHORT_LENGTH" >> "$GITHUB_OUTPUT"
```